Identity/Identity.API/Options/JwtOptions.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace Identity.API.Options;
 
-public class JWTOptions
+public class JwtOptions
 {
-    public const string JWT = "JWT";
+    public const string SectionName = "JWT";
```

Comment:
Any reason behind the name change? The [example in the documentation for the options pattern](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0) uses the name of the section as the name of the section variable as well.

Reply:
I just think this carries more meaning behind the variable, since it's explicitly used when reading the section name